src/components/plots/abstractXYPlot.ts

```diff
@@ -156,6 +157,14 @@ export module Plot {
       }
     }
 
+    /**
+     * Sets an interpolation mode.
+     */
+    public interpolate(interpolationMode: string) {
+      this._interpolationMode = interpolationMode;
+      return this;
+    }
```

Comment:
interpolate can be a function and the function argument is the most user configurable https://github.com/mbostock/d3/wiki/SVG-Shapes#line_interpolate